account_analytic_distribution/models/account_analytic_distribution_rule.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# Copyright 2017 Vicent Cubells - <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import _, fields, models
+
+
+class AccountAnalyticDistributionRule(models.Model):
```

Comment:
This should go in the same file as the account.analytic.distribution, as it's a one2many model (like sale.order.line).